config/kubernetes/base/webhook.yaml

```diff
@@ -39,6 +39,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: MY_POD_NAME
```

Comment:
How about
```suggestion
        - name: WEBHOOK_POD_NAME
```

Reply:
@jkandasa can you please handle this for both yaml

Reply:
@piyush-garg updated